### ISSUE TITLE
Error at start-up (2016.3)

### ISSUE THREAD

**Issue by Sartoric** [enriquepiatti/Magicento#208]
PhpStorm 2016.3
Magicento 3.0.2

I get it twice. Same message with slightly different error details 

`com.jetbrains.php.lang.psi.elements.ExtendsList.getReferenceElement()Lcom/jetbrains/php/lang/psi/elements/ClassReference;`

```
com.jetbrains.php.lang.psi.elements.ExtendsList.getReferenceElement()Lcom/jetbrains/php/lang/psi/elements/ClassReference;
java.lang.NoSuchMethodError: com.jetbrains.php.lang.psi.elements.ExtendsList.getReferenceElement()Lcom/jetbrains/php/lang/psi/elements/ClassReference;
	at com.magicento.models.xml.config.MagentoConfigXml.checkRewriteConflicts(MagentoConfigXml.java:189)
	at com.magicento.models.xml.config.MagentoConfigXml.checkConfigXml(MagentoConfigXml.java:80)
	at com.magicento.models.xml.config.MagentoConfigXml.getMergedXml(MagentoConfigXml.java:70)
	at com.magicento.models.xml.MagentoXml.getMergedXmlFile(MagentoXml.java:277)
	at com.magicento.MagicentoProjectComponent.getCachedXml(MagicentoProjectComponent.java:229)
	at com.magicento.MagicentoProjectComponent.getCachedConfigXml(MagicentoProjectComponent.java:177)
	at com.magicento.models.MagentoFactoryCache.findClassesForFactory(MagentoFactoryCache.java:58)
	at com.magicento.MagicentoProjectComponent.findClassesOfFactoryUri(MagicentoProjectComponent.java:480)
	at com.magicento.MagicentoProjectComponent.findClassesOfFactoryUri(MagicentoProjectComponent.java:484)
	at com.magicento.extensions.FactoryTypeProvider2.getClasses(FactoryTypeProvider2.java:1077)
	at com.magicento.extensions.FactoryTypeProvider2.getBySignature(FactoryTypeProvider2.java:891)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:469)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:313)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:79)
	at com.jetbrains.php.lang.psi.resolve.types.PhpType.globalLocationAware(PhpType.java:410)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.resolveGlobal(MemberReferenceImpl.java:147)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:143)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:140)
	at com.intellij.psi.impl.source.resolve.ResolveCache$3.compute(ResolveCache.java:142)
	at com.intellij.openapi.util.RecursionManager$2.doPreventingRecursion(RecursionManager.java:112)
	at com.intellij.psi.impl.source.resolve.ResolveCache.a(ResolveCache.java:139)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:167)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:159)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.multiResolve(PhpReferenceImpl.java:135)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.resolve(PhpReferenceImpl.java:121)
	at com.jetbrains.php.lang.annotator.PhpAnnotatorVisitor.checkAccessModifiers(PhpAnnotatorVisitor.java:418)
	at com.jetbrains.php.lang.annotator.PhpAnnotatorVisitor.visitPhpMethodReference(PhpAnnotatorVisitor.java:400)
	at com.jetbrains.php.lang.psi.elements.impl.MethodReferenceImpl.accept(MethodReferenceImpl.java:29)
	at com.jetbrains.php.lang.annotator.PhpAnnotator.annotate(PhpAnnotator.java:19)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.a(DefaultHighlightVisitor.java:139)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.visit(DefaultHighlightVisitor.java:102)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:363)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:298)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:322)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:325)
	at com.intellij.codeInsight.daemon.impl.analysis.XmlHighlightVisitor.analyze(XmlHighlightVisitor.java:748)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:325)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:325)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.analyze(DefaultHighlightVisitor.java:86)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:325)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:297)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectInformationWithProgress(GeneralHighlightingPass.java:237)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:82)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:433)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1054)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:426)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:425)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:405)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:155)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)


java.lang.NoSuchMethodError: com.jetbrains.php.lang.psi.elements.ExtendsList.getReferenceElement()Lcom/jetbrains/php/lang/psi/elements/ClassReference;
	at com.magicento.models.xml.config.MagentoConfigXml.checkRewriteConflicts(MagentoConfigXml.java:189)
	at com.magicento.models.xml.config.MagentoConfigXml.checkConfigXml(MagentoConfigXml.java:80)
	at com.magicento.models.xml.config.MagentoConfigXml.getMergedXml(MagentoConfigXml.java:70)
	at com.magicento.models.xml.MagentoXml.getMergedXmlFile(MagentoXml.java:277)
	at com.magicento.MagicentoProjectComponent.getCachedXml(MagicentoProjectComponent.java:229)
	at com.magicento.MagicentoProjectComponent.getCachedConfigXml(MagicentoProjectComponent.java:177)
	at com.magicento.models.MagentoFactoryCache.findClassesForFactory(MagentoFactoryCache.java:58)
	at com.magicento.MagicentoProjectComponent.findClassesOfFactoryUri(MagicentoProjectComponent.java:480)
	at com.magicento.MagicentoProjectComponent.findClassesOfFactoryUri(MagicentoProjectComponent.java:484)
	at com.magicento.extensions.FactoryTypeProvider2.getClasses(FactoryTypeProvider2.java:1077)
	at com.magicento.extensions.FactoryTypeProvider2.getBySignature(FactoryTypeProvider2.java:891)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:469)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:313)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:79)
	at com.jetbrains.php.lang.psi.resolve.types.PhpType.globalLocationAware(PhpType.java:410)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.resolveGlobal(MemberReferenceImpl.java:147)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:143)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:140)
	at com.intellij.psi.impl.source.resolve.ResolveCache$3.compute(ResolveCache.java:142)
	at com.intellij.openapi.util.RecursionManager$2.doPreventingRecursion(RecursionManager.java:112)
	at com.intellij.psi.impl.source.resolve.ResolveCache.a(ResolveCache.java:139)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:167)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:159)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.multiResolve(PhpReferenceImpl.java:135)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.resolve(PhpReferenceImpl.java:121)
	at com.jetbrains.php.lang.annotator.PhpAnnotatorVisitor.checkAccessModifiers(PhpAnnotatorVisitor.java:418)
	at com.jetbrains.php.lang.annotator.PhpAnnotatorVisitor.visitPhpMethodReference(PhpAnnotatorVisitor.java:400)
	at com.jetbrains.php.lang.psi.elements.impl.MethodReferenceImpl.accept(MethodReferenceImpl.java:29)
	at com.jetbrains.php.lang.annotator.PhpAnnotator.annotate(PhpAnnotator.java:19)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.a(DefaultHighlightVisitor.java:139)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.visit(DefaultHighlightVisitor.java:102)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:363)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:298)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:322)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:325)
	at com.intellij.codeInsight.daemon.impl.analysis.XmlHighlightVisitor.analyze(XmlHighlightVisitor.java:748)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:325)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:325)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.analyze(DefaultHighlightVisitor.java:86)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:325)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:297)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectInformationWithProgress(GeneralHighlightingPass.java:237)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:82)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:433)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1054)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:426)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:425)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:405)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:155)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)

```

**Comment by Sartoric** [comment]
```
com.jetbrains.php.lang.psi.elements.ExtendsList.getReferenceElement()Lcom/jetbrains/php/lang/psi/elements/ClassReference;
java.lang.NoSuchMethodError: com.jetbrains.php.lang.psi.elements.ExtendsList.getReferenceElement()Lcom/jetbrains/php/lang/psi/elements/ClassReference;
	at com.magicento.models.xml.config.MagentoConfigXml.checkRewriteConflicts(MagentoConfigXml.java:189)
	at com.magicento.models.xml.config.MagentoConfigXml.checkConfigXml(MagentoConfigXml.java:80)
	at com.magicento.models.xml.config.MagentoConfigXml.getMergedXml(MagentoConfigXml.java:70)
	at com.magicento.models.xml.MagentoXml.getMergedXmlFile(MagentoXml.java:277)
	at com.magicento.MagicentoProjectComponent.getCachedXml(MagicentoProjectComponent.java:229)
	at com.magicento.MagicentoProjectComponent.getCachedConfigXml(MagicentoProjectComponent.java:177)
	at com.magicento.models.MagentoFactoryCache.findClassesForFactory(MagentoFactoryCache.java:58)
	at com.magicento.MagicentoProjectComponent.findClassesOfFactoryUri(MagicentoProjectComponent.java:480)
	at com.magicento.MagicentoProjectComponent.findClassesOfFactoryUri(MagicentoProjectComponent.java:484)
	at com.magicento.extensions.FactoryTypeProvider2.getClasses(FactoryTypeProvider2.java:1077)
	at com.magicento.extensions.FactoryTypeProvider2.getBySignature(FactoryTypeProvider2.java:891)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:469)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:313)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:79)
	at com.jetbrains.php.lang.psi.resolve.types.PhpType.globalLocationAware(PhpType.java:410)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.resolveGlobal(MemberReferenceImpl.java:147)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:143)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:140)
	at com.intellij.psi.impl.source.resolve.ResolveCache$3.compute(ResolveCache.java:142)
	at com.intellij.openapi.util.RecursionManager$2.doPreventingRecursion(RecursionManager.java:112)
	at com.intellij.psi.impl.source.resolve.ResolveCache.a(ResolveCache.java:139)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:167)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:159)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.multiResolve(PhpReferenceImpl.java:135)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.resolve(PhpReferenceImpl.java:121)
	at com.jetbrains.php.lang.PhpCallbackFunctionUtil.isFunctionCallbackFunction(PhpCallbackFunctionUtil.java:85)
	at com.jetbrains.php.lang.PhpReferenceContributor$CallbackReferenceProvider.getCallbackReference(PhpReferenceContributor.java:69)
	at com.jetbrains.php.lang.PhpReferenceContributor$CallbackReferenceProvider.getReferencesByElement(PhpReferenceContributor.java:104)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistryImpl.doGetReferencesFromProviders(ReferenceProvidersRegistryImpl.java:120)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistry.getReferencesFromProviders(ReferenceProvidersRegistry.java:66)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistry.getReferencesFromProviders(ReferenceProvidersRegistry.java:60)
	at com.jetbrains.php.lang.psi.elements.impl.StringLiteralExpressionImpl.getReferences(StringLiteralExpressionImpl.java:283)
	at org.intellij.plugins.intelliLang.references.InjectedReferencesContributor.getInjectedReferences(InjectedReferencesContributor.java:56)
	at org.intellij.plugins.intelliLang.references.InjectedReferencesInspection$1.visitElement(InjectedReferencesInspection.java:40)
	at com.jetbrains.php.lang.psi.elements.impl.StringLiteralExpressionImpl.accept(StringLiteralExpressionImpl.java:29)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:81)
	at com.intellij.codeInspection.InspectionEngine.createVisitorAndAcceptElements(InspectionEngine.java:69)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.a(LocalInspectionsPass.java:280)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.a(LocalInspectionsPass.java:249)
	at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:123)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1048)
	at com.intellij.concurrency.ApplierCompleter.b(ApplierCompleter.java:96)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:106)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:91)
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.pollAndExecCC(ForkJoinPool.java:1190)
	at java.util.concurrent.ForkJoinPool.helpComplete(ForkJoinPool.java:1879)
	at java.util.concurrent.ForkJoinPool.awaitJoin(ForkJoinPool.java:2045)
	at java.util.concurrent.ForkJoinTask.doJoin(ForkJoinTask.java:390)
	at java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:719)
	at java.util.concurrent.ForkJoinPool.invoke(ForkJoinPool.java:2616)
	at com.intellij.concurrency.JobLauncherImpl.invokeConcurrentlyUnderProgress(JobLauncherImpl.java:61)
	at com.intellij.concurrency.JobLauncher.invokeConcurrentlyUnderProgress(JobLauncher.java:57)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.a(LocalInspectionsPass.java:251)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.a(LocalInspectionsPass.java:222)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.collectInformationWithProgress(LocalInspectionsPass.java:131)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:82)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:433)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1054)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:426)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:425)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:405)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:155)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)


java.lang.NoSuchMethodError: com.jetbrains.php.lang.psi.elements.ExtendsList.getReferenceElement()Lcom/jetbrains/php/lang/psi/elements/ClassReference;
	at com.magicento.models.xml.config.MagentoConfigXml.checkRewriteConflicts(MagentoConfigXml.java:189)
	at com.magicento.models.xml.config.MagentoConfigXml.checkConfigXml(MagentoConfigXml.java:80)
	at com.magicento.models.xml.config.MagentoConfigXml.getMergedXml(MagentoConfigXml.java:70)
	at com.magicento.models.xml.MagentoXml.getMergedXmlFile(MagentoXml.java:277)
	at com.magicento.MagicentoProjectComponent.getCachedXml(MagicentoProjectComponent.java:229)
	at com.magicento.MagicentoProjectComponent.getCachedConfigXml(MagicentoProjectComponent.java:177)
	at com.magicento.models.MagentoFactoryCache.findClassesForFactory(MagentoFactoryCache.java:58)
	at com.magicento.MagicentoProjectComponent.findClassesOfFactoryUri(MagicentoProjectComponent.java:480)
	at com.magicento.MagicentoProjectComponent.findClassesOfFactoryUri(MagicentoProjectComponent.java:484)
	at com.magicento.extensions.FactoryTypeProvider2.getClasses(FactoryTypeProvider2.java:1077)
	at com.magicento.extensions.FactoryTypeProvider2.getBySignature(FactoryTypeProvider2.java:891)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:469)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:313)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:79)
	at com.jetbrains.php.lang.psi.resolve.types.PhpType.globalLocationAware(PhpType.java:410)
	at com.jetbrains.php.lang.psi.elements.impl.MemberReferenceImpl.resolveGlobal(MemberReferenceImpl.java:147)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:143)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl$1.resolve(PhpReferenceImpl.java:140)
	at com.intellij.psi.impl.source.resolve.ResolveCache$3.compute(ResolveCache.java:142)
	at com.intellij.openapi.util.RecursionManager$2.doPreventingRecursion(RecursionManager.java:112)
	at com.intellij.psi.impl.source.resolve.ResolveCache.a(ResolveCache.java:139)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:167)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:159)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.multiResolve(PhpReferenceImpl.java:135)
	at com.jetbrains.php.lang.psi.elements.impl.PhpReferenceImpl.resolve(PhpReferenceImpl.java:121)
	at com.jetbrains.php.lang.PhpCallbackFunctionUtil.isFunctionCallbackFunction(PhpCallbackFunctionUtil.java:85)
	at com.jetbrains.php.lang.PhpReferenceContributor$CallbackReferenceProvider.getCallbackReference(PhpReferenceContributor.java:69)
	at com.jetbrains.php.lang.PhpReferenceContributor$CallbackReferenceProvider.getReferencesByElement(PhpReferenceContributor.java:104)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistryImpl.doGetReferencesFromProviders(ReferenceProvidersRegistryImpl.java:120)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistry.getReferencesFromProviders(ReferenceProvidersRegistry.java:66)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistry.getReferencesFromProviders(ReferenceProvidersRegistry.java:60)
	at com.jetbrains.php.lang.psi.elements.impl.StringLiteralExpressionImpl.getReferences(StringLiteralExpressionImpl.java:283)
	at org.intellij.plugins.intelliLang.references.InjectedReferencesContributor.getInjectedReferences(InjectedReferencesContributor.java:56)
	at org.intellij.plugins.intelliLang.references.InjectedReferencesInspection$1.visitElement(InjectedReferencesInspection.java:40)
	at com.jetbrains.php.lang.psi.elements.impl.StringLiteralExpressionImpl.accept(StringLiteralExpressionImpl.java:29)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:81)
	at com.intellij.codeInspection.InspectionEngine.createVisitorAndAcceptElements(InspectionEngine.java:69)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.a(LocalInspectionsPass.java:280)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.a(LocalInspectionsPass.java:249)
	at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:123)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1048)
	at com.intellij.concurrency.ApplierCompleter.b(ApplierCompleter.java:96)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:106)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:91)
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.pollAndExecCC(ForkJoinPool.java:1190)
	at java.util.concurrent.ForkJoinPool.helpComplete(ForkJoinPool.java:1879)
	at java.util.concurrent.ForkJoinPool.awaitJoin(ForkJoinPool.java:2045)
	at java.util.concurrent.ForkJoinTask.doJoin(ForkJoinTask.java:390)
	at java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:719)
	at java.util.concurrent.ForkJoinPool.invoke(ForkJoinPool.java:2616)
	at com.intellij.concurrency.JobLauncherImpl.invokeConcurrentlyUnderProgress(JobLauncherImpl.java:61)
	at com.intellij.concurrency.JobLauncher.invokeConcurrentlyUnderProgress(JobLauncher.java:57)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.a(LocalInspectionsPass.java:251)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.a(LocalInspectionsPass.java:222)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.collectInformationWithProgress(LocalInspectionsPass.java:131)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:82)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:433)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1054)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:426)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:425)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:405)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:155)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)



```

**Comment by enriquepiatti** [comment]
This error is fixed, I will release the new version (v3.1.0) with the bugfix and some extra features in a few days

**Comment by Sartoric** [comment]
Good :)
Thanks
